src/DOMupdates.js

```diff
@@ -1,18 +1,38 @@
+import Player from "./Player";
```

Comment:
What is the need to import the `Player` class in this file? Are you instantiating players in this file? Otherwise, it shouldn't be needed.